<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; font-style:italic;&quot;&gt;Lista Linear Simplesmente Encadeada&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Valor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lista:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaida">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>371</width>
      <height>177</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;INSERIR&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ACESSAR&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RETIRAR&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonIncluirInicio">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Inserir no Inicio</string>
           </property>
           <property name="text">
            <string>  II</string>
           </property>
           <property name="icon">
            <iconset resource="ArquivosDeRecursos.qrc">
             <normaloff>:/new/prefix/imagens/Arquivos/incluir.jpg</normaloff>:/new/prefix/imagens/Arquivos/incluir.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAcessarInicio">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Acessar no Inicio</string>
           </property>
           <property name="text">
            <string>  AI</string>
           </property>
           <property name="icon">
            <iconset resource="ArquivosDeRecursos.qrc">
             <normaloff>:/new/prefix/imagens/Arquivos/acessar.jpg</normaloff>:/new/prefix/imagens/Arquivos/acessar.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRetiraInicio">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Retirar no Inicio</string>
           </property>
           <property name="text">
            <string>  RI</string>
           </property>
           <property name="icon">
            <iconset resource="ArquivosDeRecursos.qrc">
             <normaloff>:/new/prefix/imagens/Arquivos/retirar.jpg</normaloff>:/new/prefix/imagens/Arquivos/retirar.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButtonIncluirFinal">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Inserir no Final</string>
           </property>
           <property name="text">
            <string>  IF</string>
           </property>
           <property name="icon">
            <iconset resource="ArquivosDeRecursos.qrc">
             <normaloff>:/new/prefix/imagens/Arquivos/incluir.jpg</normaloff>:/new/prefix/imagens/Arquivos/incluir.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAcessarFinal">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Acessar no Final</string>
           </property>
           <property name="text">
            <string>  AF</string>
           </property>
           <property name="icon">
            <iconset resource="ArquivosDeRecursos.qrc">
             <normaloff>:/new/prefix/imagens/Arquivos/acessar.jpg</normaloff>:/new/prefix/imagens/Arquivos/acessar.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRetiraFinal">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Retirar no Final</string>
           </property>
           <property name="text">
            <string>  RF</string>
           </property>
           <property name="icon">
            <iconset resource="ArquivosDeRecursos.qrc">
             <normaloff>:/new/prefix/imagens/Arquivos/retirar.jpg</normaloff>:/new/prefix/imagens/Arquivos/retirar.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButtonIncluirMeio">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Inserir na Posição</string>
         </property>
         <property name="text">
          <string>  IP</string>
         </property>
         <property name="icon">
          <iconset resource="ArquivosDeRecursos.qrc">
           <normaloff>:/new/prefix/imagens/Arquivos/incluir.jpg</normaloff>:/new/prefix/imagens/Arquivos/incluir.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAcessarMeio">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Acessar na Posição</string>
         </property>
         <property name="text">
          <string>  AP</string>
         </property>
         <property name="icon">
          <iconset resource="ArquivosDeRecursos.qrc">
           <normaloff>:/new/prefix/imagens/Arquivos/acessar.jpg</normaloff>:/new/prefix/imagens/Arquivos/acessar.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRetiraMeio">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Retirar na Posição</string>
         </property>
         <property name="text">
          <string>  RP</string>
         </property>
         <property name="icon">
          <iconset resource="ArquivosDeRecursos.qrc">
           <normaloff>:/new/prefix/imagens/Arquivos/retirar.jpg</normaloff>:/new/prefix/imagens/Arquivos/retirar.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quant. Elementos</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaidaQtdElementos">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsercaoOrdenada">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>118</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Inserir Ordenado</string>
    </property>
    <property name="text">
     <string>  IO</string>
    </property>
    <property name="icon">
     <iconset resource="ArquivosDeRecursos.qrc">
      <normaloff>:/new/prefix/imagens/Arquivos/incluir.jpg</normaloff>:/new/prefix/imagens/Arquivos/incluir.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOrdenar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>141</width>
      <height>37</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Ordenar</string>
    </property>
    <property name="text">
     <string>  ORDENAR</string>
    </property>
    <property name="icon">
     <iconset resource="ArquivosDeRecursos.qrc">
      <normaloff>:/new/prefix/imagens/Arquivos/ordenacao.jpg</normaloff>:/new/prefix/imagens/Arquivos/ordenacao.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>415</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ArquivosDeRecursos.qrc"/>
 </resources>
 <connections/>
</ui>
